docs/podman.1.md

```diff
@@ -39,6 +39,11 @@ Path to where the cpu performance results should be written
 
 log messages above specified level: debug, info, warn, error (default), fatal or panic
 
+**--namespace**
```

Comment:
We should put together a blog and/or tutorial about this at some point in time, not for this PR  though.

Reply:
Agree, I think this is a good blog